service/tokenizer.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 The casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"sort"
+	"unicode/utf8"
+
+	"github.com/huichen/sego"
+)
+
+var segmenter sego.Segmenter
+
+type wordfreq struct {
+	word string
+	num  int
+}
+
+func init() {
+	segmenter.LoadDictionary("./dictionary/dictionary.txt")
```

Comment:
Is dictionary.txt in this repo?

Reply:
The dictionary used  in previous projects, and sego has also used them before